### ISSUE TITLE
Add examples to the README and website?

### ISSUE THREAD

**Issue by maelle** [sckott/cchecksbadges#1]
Examples where one would actually see the badge :-)

**Comment by sckott** [comment]
good idea, will do

**Comment by sckott** [comment]
Is this what you had in mind https://github.com/r-hub/cchecksbadges#badges ?

**Comment by maelle** [comment]
Yes, thank you! :star_struck:  Could you please add alternative text as well?
